app/models/company.py

```diff
@@ -95,3 +97,16 @@ def retrieve_authorized_clients(self):
                 ),
             )
         )
+
+    @cached_property
+    def is_certified(self):
+        today = date.today()
+        for company_certification in self.certifications:
+            if (
+                company_certification.attribution_date
+                <= today
```

Comment:
:orange_circle: `company_certification.attribution_date <= today` is still true, isn't it?